axon-server/administration/multi-context.md

```diff
@@ -2,17 +2,138 @@
 
 > Note: This feature is only available on the Enterprise Edition of AxonServer
 
-You can use a single AxonServer \(cluster\) to store events for multiple bounded contexts. Each context will have its own set of files \(stored in a separate directory\). Each context may have a different master in an AxonServer cluster.‌
+To recap the definition as we have seen in the [clustering](../installation/axon-server-clustering.md) section, contexts allow for strong segregation of data without requiring deploying and managing full instances. 
 
-Creating a new context is done using the user interface, or via the command line interface:
+An Axon Server EE cluster can be setup to store events for multiple contexts. Each context has it own set of files \(containing Event/Snapshot data\) stored in a separate directory. The creation process involves the addition of member nodes within a cluster that will serve that context and the provisioning of a separate physical directory on each of those member nodes. The location of the newly created directory would be under the _root data directory_ of the member node.
+
+A depiction of multiple registered contexts within an Axon Server EE cluster is shown below
+
+![Multiple contexts within an Axon Server EE cluster](../../.gitbook/assets/multi-context.jpg)
+
+The [clustering](../installation/axon-server-clustering.md) section details the creation of the \__admin and default contexts_ when a new Axon Server cluster is created. The _\_admin_ context __is used to process all configuration changes in Axon Server, so it contains the master configuration from which all contexts get their information. The __\__admin_ context cannot be deleted and thus not visible under the data directory. The _default_ context is the context used by clients when they have not specified any context information. In case you would like to create a cluster without creating a default context, it is recommended to use the [Automatic-Initialization](../installation/axon-server-clustering.md#automatic-initialization) feature where you can control explicitly which contexts can be created or not.
```

Comment:
`context cannot be deleted and thus _is_ not visible under the data directory.` i.o. `context cannot be deleted and thus not visible under the data directory.`

Reply:
The _admin context does not have an event store. Therefore there is no _admin directory under the data directory.